packages/react-components/src/label/src/label.jsx

```diff
@@ -176,6 +180,7 @@ export function PureLabel(props) {
         const classes = mergeClasses(
             naked && "naked",
             highlight && "highlight",
+            disabled && "disabled",
```

Comment:
You don't need this since the tag component is using the SUI Label and not this custom one.